repo2docker/buildpacks/conda/install-miniconda.bash

```diff
@@ -21,7 +21,7 @@ if ! echo "${MD5SUM}  ${INSTALLER_PATH}" | md5sum  --quiet -c -; then
 fi
 
 bash ${INSTALLER_PATH} -b -p ${CONDA_DIR}
-export PATH="${CONDA_DIR}/bin:$PATH"
+. ${CONDA_DIR}/etc/profile.d/conda.sh
```

Comment:
This is a good thing to do, but it would also probably be good to add a link to this file from `/etc/profile.d/conda.sh` so that any full shell invokes it as well.